STX.REST.RESTFulSense.Clients/Services/Foundations/HttpExchanges/HttpExchangeService.Validations.cs

```diff
@@ -0,0 +1,145 @@
+// ----------------------------------------------------------------------------------
+// Copyright (c) The Standard Organization: A coalition of the Good-Hearted Engineers
+// ----------------------------------------------------------------------------------
+
+using System;
+using System.Net;
+using System.Net.Http;
+using STX.REST.RESTFulSense.Clients.Models.Services.HttpExchanges;
+using STX.REST.RESTFulSense.Clients.Models.Services.HttpExchanges.Exceptions;
+
+namespace STX.REST.RESTFulSense.Clients.Services.Foundations.HttpExchanges
+{
+    internal partial class HttpExchangeService
+    {
+        private static void ValidateOnGet(
+            HttpExchange httpExchange,
+            HttpMethod defaultHttpMethod,
+            Version defaultHttpVersion,
+            HttpVersionPolicy defaultHttpVersionPolicy)
+        {
+            ValidateHttpExchange(httpExchange);
```

Comment:
I think, this should be inside ValidateHttpExchangeRequest

![image](https://github.com/The-Standard-Organization/STX.REST.RESTFulSense.Clients/assets/44029921/e6926ec4-a722-439a-97c5-5a5e70417517)